creator-node/src/utils.js

```diff
@@ -7,7 +7,7 @@ const stream = require('stream')
 const retry = require('async-retry')
 const { promisify } = require('util')
 const pipeline = promisify(stream.pipeline)
-const { logger: genericLogger } = require('./logging.js')
+const { logger } = require('./logging')
```

Comment:
just curious why make these 2 changes? (remove genericLogger rename, and .js extension)

Reply:
I removed the `.js` part to match with `const { logger } = require('./logging')` which is found in the other files.

I removed the genericLogger rename in order to replace `console` with `logger` on line 27 since that pattern matches the other files I was looking at.

Should I perhaps change line 27 alone to be `genericLogger.info()`?

Reply:
Nvm! I misunderstood the syntax. Let me update this after a rebase. Thanks for pointing this out!

Reply:
ok cool! i'll re-review after